docs/modules/ROOT/pages/vshn-managed/postgresql/security.adoc

```diff
@@ -55,3 +55,30 @@ spec:
 
 <1> List of groups to be allowed limited access to the PostgreSQL namespace
 <2> List of users to be allowed limited access to the PostgreSQL namespace
+
+
+== Network Policies
+
+By default, PostgreSQL is not accessible from outside the cluster.
+To allow access from outside the cluster, a .spec.parameters.security.network must be configured, for example:
```

Comment:
I don't see you configuring `.spec.parameters.security.network ` in the example you propose.